modules/appeals_api/app/sidekiq/appeals_api/clean_up_pii.rb

```diff
@@ -3,21 +3,15 @@
 require 'sidekiq'
 
 module AppealsApi
-  class SupplementalClaimCleanUpPii
+  class CleanUpPii
```

Comment:
I'm not sure this is actually what we want to do. I'm thinking that calling this method three times in the job won't be a big deal because it's basically just a big query + an update statement, but I'm very much open to other perspectives.

Reply:
Since we're talking a fairly low volume of records, I'm not too concerned about this change. To be more confident, you could potentially review the job runtime for all three jobs prior to merging, or monitor the new, single job's runtime after this change is deployed, to ensure that it doesn't exceed a minute or so of runtime.